detox/src/devices/Device.js

```diff
@@ -23,7 +34,52 @@ class Device {
     await this.relaunchApp({delete: !argparse.getArgValue('reuse')});
   }
 
+  async _copyArtifacts() {
```

Comment:
I think this is not the job of `Device` to move artifacts around. Why not merge all the logic in an `ArtifactsHelper` as you previously suggested ?